compiler-rt/lib/rtsan/tests/CMakeLists.txt

```diff
@@ -94,7 +94,6 @@ foreach(arch ${RTSAN_TEST_ARCH})
     COMPILE_DEPS ${RTSAN_UNITTEST_HEADERS}
     SOURCES ${RTSAN_NOINST_TEST_SOURCES}
             ${COMPILER_RT_GOOGLETEST_SOURCES}
```

Comment:
Will you remove this dependency in the commented out line around 60 as well? Just want to make sure we don't repeat this mistake when that is commented out